src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/BaseFunctionalTest.java

```diff
@@ -31,4 +33,14 @@ public void setUp() throws Exception {
         RestAssured.useRelaxedHTTPSValidation();
     }
 
+    public void cleanUp(String taskId, String token) {
+        given()
+            .header(SERVICE_AUTHORIZATION, token)
+            .contentType(APPLICATION_JSON_VALUE)
+            .baseUri(camundaUrl)
+            .basePath("/task/" + taskId + "/complete")
+            .when()
+            .post();
+    }
```

Comment:
Can we check the state of the completed task? I.e. check that the task no longer exists as part of completing and clean up.